docs/standard/serialization/system-text-json/customize-properties.md

```diff
@@ -271,83 +266,19 @@ The following example shows you how to use Copilot to modify existing code to cu
    ```
 
 1. In Copilot Chat, enter the following prompt to modify the code to customize names and order of the JSON serialization output.
+   
+    ```copilot-prompt
+   `#Example.cs` modify code to use `System.Text.Json` to customize property names and order of JSON output from serialization.
```

Comment:
Why do we need backticks for the file name and namespace? The entire prompt is already code-fenced so not needed for localization.

Reply:
leftover from when it was text